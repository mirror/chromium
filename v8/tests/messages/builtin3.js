// See if we can't access natives by tricking the parser.
function foo() {
  x = 0 %StringEquals('a', 'b');
  return x;
}


foo();
