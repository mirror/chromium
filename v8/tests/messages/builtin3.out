builtin3.js:3: Uncaught ReferenceError: StringEquals is not defined
  x = 0 %StringEquals('a', 'b');
         ^
